lib/dealership.rb

```diff
@@ -0,0 +1,54 @@
+class Dealership
+  attr_reader :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+    @has_inventory = false
```

Comment:
You will not want to have this here either. You will do the logic in your `has_inventory?` method below.